src/hashstore/hashstore_factory.py

```diff
@@ -0,0 +1,52 @@
+"""Core module for HashStore Factory"""
+import importlib
+
+
+class HashStoreFactory:
+    """A factory class for creating `HashStore`-like objects (classes
+    that implement the 'HashStore' abstract methods)
+
+    This factory class provides a method to retrieve a `HashStore` object
+    based on a given module (ex. "hashstore.filehashstore.filehashstore")
+    and class name (ex. "FileHashStore").
+    """
+
+    @staticmethod
+    def get_hashstore(module_name, class_name, properties=None):
+        """Get a `HashStore`-like object based on the specified `module_name` and `class_name`.
```

Comment:
>  a `HashStore`-like

I think you told me backticks in docs produced unexpected side-effects? Checking if this was an oversight, or on purpose?

Reply:
Nope, not an oversight - it was actually an issue with git commit messages where the contents of a `...` would disappear and be blank.

Reply:
my bad - sorry